diff -Nuar libjpeg-turbo-1.3.1.orig/CMakeLists.txt libjpeg-turbo-1.3.1/CMakeLists.txt
--- libjpeg-turbo-1.3.1.orig/CMakeLists.txt	2014-03-11 07:30:21.000000000 +0100
+++ libjpeg-turbo-1.3.1/CMakeLists.txt	2014-09-13 16:27:30.490320900 +0200
@@ -96,18 +96,6 @@
   message(STATUS "In-memory source/destination managers disabled")
 endif()
 
-if(MSVC)
-  # Use the static C library for all build types
-  foreach(var CMAKE_C_FLAGS CMAKE_C_FLAGS_DEBUG CMAKE_C_FLAGS_RELEASE
-    CMAKE_C_FLAGS_MINSIZEREL CMAKE_C_FLAGS_RELWITHDEBINFO)
-    if(${var} MATCHES "/MD")
-      string(REGEX REPLACE "/MD" "/MT" ${var} "${${var}}")
-    endif()
-  endforeach()
-
-  add_definitions(-W3 -wd4996)
-endif()
-
 # Detect whether compiler is 64-bit
 if(MSVC AND CMAKE_CL_64)
   set(SIMD_X86_64 1)
